Makefile

```diff
@@ -78,8 +78,11 @@ install: $(binary)
 $(tests_binary): $(GEN_FILES) $(SOURCE_FILES) $(TEST_FILES) | $(BUILD_DIR)
 	${PONYC} $(arch_arg) $(LINKER) --debug -o ${BUILD_DIR} $(SRC_DIR)/test
 
+integration: $(binary) $(tests_binary)
+	CORRAL_BIN=$$(pwd)/$(binary) $(tests_binary) --only=integration --sequential
```

Comment:
what's going on with CORRAL_BIN here?

Reply:
This is the same integration setup that pony-stable uses. It just sets an env var to point to the built corral binary that the integration test should execute.

Reply:
ok, I'll review it in that context.

Looking at it, I think we can make it more robust. I'm not sure if the setup you have will respect config=??? setup. I will verify that is the case.

Reply:
It worked for me when I just tried it. Since $(binary) is the current config, the integration tests will use that,